### PR TITLE
bloaty 1.0

### PR DESCRIPTION
**Pull request by zlern2k**
- [Y] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [Y] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [Y] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [Y] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Bloaty was asked for in https://github.com/Homebrew/homebrew-core/issues/11803; now that it has a release version on github, I'm submitting this formula because I need it, too.

### CODE REVIEW

**Comment by zlern2k** [comment]
Bah. Moved bloaty into the bin directory and then forgot to re-run 'brew test' before submitting. figuring out how to update the pull request now...

File: Formula/bloaty.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Bloaty < Formula
+  desc "Size profiler for binaries"
+  homepage "https://github.com/google/bloaty"
+  url "https://github.com/google/bloaty/releases/download/v1.0/bloaty-1.0.tar.bz2"
+  sha256 "e1cf9830ba6c455218fdb50e7a8554ff256da749878acfaf77c032140d7ddde0"
+  depends_on "cmake" => :build
```
Comment (DomT4) [inline Formula/bloaty.rb]: New line between `sha256` and the first `depends_on` please.

File: Formula/bloaty.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "-j#{ENV.make_jobs}"
```
Comment (DomT4) [inline Formula/bloaty.rb]: Drop the ` "-j#{ENV.make_jobs}"`.

File: Formula/bloaty.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    bin.install buildpath/"bloaty"
```
Comment (DomT4) [inline Formula/bloaty.rb]: Is there no `make install`?

**Review by DomT4 — CHANGES_REQUESTED** [review]

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/65d8212b3073522ba7717ea2c34a3fc147ef09c6. Thank you for your contribution to Homebrew @zlern2k; we appreciate it! :smiley_cat:

**Pull request closed without merging**